android/app/src/main/res/values-v26/styles.xml

```diff
@@ -0,0 +1,8 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
+
+    <!-- App Theme for Android 8.0 and above -->
+    <style name="AppTheme" parent="BaseAppTheme">
+        <item name="android:autofilledHighlight">@android:color/transparent</item>
+    </style>
+
+</resources>
```

Comment:
The autofilled bg was introduced in Android 8.0 (API 26). We can't add it to the common styles file as our minimum Android version is 5.0 (API 21), hence the need for this API-26 specific styles file.